### PR TITLE
[DOCFIX] add server configuration docs

### PR DESCRIPTION
**Pull request by LuQQiu**
This PR should be cherry-pick to 1.8

### CODE REVIEW

**Comment by LuQQiu** [comment]
@aaudiber  Added the documentation for Server configuration checker, PTAL, thanks!

File: docs/en/Getting-Started-on-Cluster.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ you plan to run Alluxio Master on. Let's name it Alluxio Master node, and assume
 $ echo "alluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}" >> conf/alluxio-site.properties
 ```
 
+Server-side configuration checker helps discover configuration errors and warnings.
```
Comment (aaudiber) [inline docs/en/Getting-Started-on-Cluster.md]: could we move this to the end of the "Start Alluxio" section? The checker requires Alluxio to be running, so I think it makes sense to bring it up at the time that they could use it. We can suggest going to the web UI and verifying that the configuration check passes.

Comment (LuQQiu) [reply docs/en/Getting-Started-on-Cluster.md]: I first think of putting it under 
```
$ bin/alluxio-start.sh master

Congratulations! Alluxio master is now up and running! You can visit
http://ALLUXIO_MASTER_HOSTNAME:19999 to see the status of the Alluxio master.
```

but  if users have configuration errors or warnings, those errors/warnings would show on the web UI directly. We would not need to ducument it here.

So I just remove the mention of checker in Getting-started-on-cluster.

**Comment by LuQQiu** [comment]
@aaudiber PTAL, thanks! removed the mention in getting start in cluster, because we already point them to the web UI, if users have any configuration errors, they should see them on the web UI.

**Pull request merged**